web/app/templates/base.html

```diff
@@ -30,6 +31,16 @@
   {{ moment.include_moment() }}
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
   <script type="text/javascript">
+  
     var $SCRIPT_ROOT = {{ url_for('main.index', _external = True)|tojson|safe }}
+    var csrftoken = $('meta[name=csrf-token]').attr('content')
```

Comment:
![csrf](https://cloud.githubusercontent.com/assets/6125097/15386774/e7c56cc8-1da9-11e6-8cfe-e7fd3652a597.png)

Should this really be a global variable? Wouldn't this allow extrenal scripts we pull in (bootstrap, jquery) to easily abuse this variable and bypass the point of this feature?

External scripts may come from trusted sources, but they should never be trusted to see our app-variables.

Maybe we could protect this using a closure (a wrapper function)?